client/src/components/Nav Menu/NavMenu.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import './NavMenu.css'
+
+class NavMenu extends Component {
```

Comment:
This component doesn't have any state, so can you please make it a function component?